desk

```diff
@@ -114,56 +114,95 @@ cmd_run() {
 }
 
 
+# Usage:        desk list [options]
+# Description:  List all desks along with a description
+# --only-names: List only the names of the desks
+# --no-format:  Use ' - ' to separate names from descriptions
```

Comment:
Ah, no more `usage()` message? I kinda liked that. 

We can implement in a follow-up PR, I guess.